src/main/native/cpp/analysis/FilteringUtils.cpp

```diff
@@ -10,7 +10,11 @@
 #include <frc/LinearFilter.h>
 #include <frc/MedianFilter.h>
 #include <units/math.h>
+#include <wpi/StringRef.h>
```

Comment:
StringRef is used in FilteringUtils.h as well, so this include should be moved there.